# PlanningSrv.srv
string target_position
---
bool success
